README.md

```diff
@@ -21,7 +21,7 @@
 
 # 📖OOP school library <a name="about-project"></a>
 
-**OOP school library** In this project, I create a form of UI for your school library app. This way it can be invoked as an executable and not something you use in IRB exclusively.
+**OOP school library** In this project, the main focus is on refactoring the existing codebase to improve its maintainability, readability, and adherence to best practices.
 
 ## 🛠 Built With <a name="built-with"></a>
```

Comment:
- [ ] Following the ``Single responsibility rule``, the validation and checking of the persons type should be placed on a different method, which should in turn call the `create_student` or `create_teacher` method depending on the option selected, I'll suggest you update your code to follow this guideline. :+1: :+1: :100: 

- - Code section screenshot:
![Selection_003](https://github.com/Gardimy/OOP-school-library/assets/107632780/ade5a0d2-5b39-4ce9-a0cf-050bdf73e2e1)

- [ ] Also while creating a person, you didn't provide a prompt to take the person's name, I'll suggest you update that also as all the person tends to coemup with the default name of `unknown`. :+1: :+1: :small_airplane: